api/lttb.md

```diff
@@ -0,0 +1,39 @@
+# lttb()  <tag type="toolkit">Toolkit</tag><tag type="experimental">Experimental</tag>
+[Largest Triangle Three Buckets](https://github.com/sveinn-steinarsson/flot-downsample)
+is a downsampling method that tries to retain visual similarity between the
+downsampled data and the original dataset. TimescaleDB Toolkit provides an
+implementation of this which takes `(timestamp, value)` pairs, sorts them if
+needed, and downsamples them.
+
+## Required Arguments
+|Name| Type |Description|
+|---|---|---|
+| `time` | `TIMESTAMPTZ` | Time (x) value for the data point. |
+| `value` | `DOUBLE PRECISION` |  Data (y) value for the data point. |
+| `resolution` | `INTEGER` | Number of points the output should have. |
+
+## Returns
+
+|Column|Type|Description|
+|---|---|---|
+| `sortedtimevector` | `SortedTimevector` | A [`timevector`][hyperfunctions-timevectors] object containing the downsampled points.  It can be unpacked via `unnest` |
```

Comment:
```suggestion
|Column|Type|Description|
|-|-|-|
|`sortedtimevector`|`SortedTimevector`|A [`timevector`][hyperfunctions-timevectors] object containing the downsampled points.  It can be unpacked via `unnest`.|
```